src/index.html

```diff
@@ -18,7 +18,6 @@
 		<meta name="theme-color" content="#ffffff" />
 
 		<link rel="manifest" href="/build/public/manifest.json" />
-		<script>window._METIS_RUNTIME_CONFIG = {};</script>
```

Comment:
@pyoner is it okay, that we got rid of the runtime config feature? I think, `METIS_RUNTIME_CONFIG` was introduced by @knopki

Reply:
This commit **breaks** the compatibility with `METIS_RUNTIME_CONFIG`!
If you need this feature, I must revert the commit and write code to include the runtime config in the app.
But I don't see any case where this feature would be useful.

Reply:
@pyoner the point of `METIS_RUNTIME_CONFIG` is that we can configure the production build without the re-compilation. As for now, we can live without it, but it seems to be a comfortable feature :thinking:

Reply:
> @pyoner the point of `METIS_RUNTIME_CONFIG` is that we can configure the production build without the re-compilation. As for now, we can live without it, but it seems to be a comfortable feature thinking

ok, I'll fix it

Reply:
Thank you in advance! Much appreciated